src/components/molecules/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -78,17 +83,22 @@ export const HamburgerMenu = ({
 }: {
     links: { label: string; link: string }[];
 }) => {
-    const [showMore, setShowMore] = useState(false);
+    const [showMore, setShowMore] = useState<boolean>(false);
+    const [showTradingSublinks, setShowTradingSublinks] =
+        useState<boolean>(false);
 
     return (
         <Menu>
-            {({ close }) => (
+            {({ open, close }) => (
                 <>
                     <Menu.Button aria-label='Hamburger Menu'>
-                        <Burger className='h-6 w-6 tablet:h-8 tablet:w-8' />
+                        <Burger className='h-6 w-6 text-neutral-400' />
                     </Menu.Button>
+                    {open && (
```

Comment:
When mobile menu is open in tablet and only taking up partial width of the browser, we are showing an overlay:

https://www.figma.com/design/YIBsk1ihFbPlDb8XldGj1T/SF-Design-System?node-id=691-11909&m=dev